backends/arm/_passes/arm_pass_manager.py

```diff
@@ -167,9 +169,14 @@ def _tosa_INT_pipeline(self, exported_program: ExportedProgram) -> GraphModule:
 
         self.add_pass(FuseViewCopyTransform())
         self.add_pass(FuseConstantArgsPass(exported_program))
+        self.add_pass(InsertTableOpsPass(exported_program))
+        # If we have a conv2d with int16 activation split up into a convolution
+        # and an addition, to work-around the lack of support for int48 in torch
```

Comment:
> # and an addition, to work-around the lack of support for int48 in torch
Can it be done by using torch.dtype.int64 instead and then detecting and lowering it as int48 downstream?